src/components/hooks/useMeetingDestination.js

```diff
@@ -1,5 +1,6 @@
 import {useEffect, useContext, useState} from 'react';
 import {concatMap} from 'rxjs/operators';
+import {MeetingState} from '@webex/component-adapter-interfaces';
```

Comment:
@alinasuciu please revert the changes in this file, they are done by @lungancatalin in a different PR